ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -54,29 +54,31 @@ export function showErrorMessages(errors: any) {
 }
 
 export function getFilterErrors(filters: any) {
+  return _getFilterErrors(filters, false)
+}
+
+export function getSearchFormFilterErrors(filters: any) {
+  return _getFilterErrors(filters, true)
+}
+
+function _getFilterErrors(filters: any, allowEmptyGeo: boolean) {
   const errors = new Set()
   let geometryErrors = new Set<string>()
   let dateErrors = new Set<string>()
   for (let i = 0; i < filters.length; i++) {
     const filter = filters[i]
-    getGeometryErrors(filter).forEach(msg => {
+    getGeometryErrors(filter, allowEmptyGeo).forEach(msg => {
```

Comment:
Should this be limited to geos?  Seems all fields would suffer this same fate, and we would want to allow empties for numbers, dates, enumerations, etc.

Reply:
The only other validation is for date so it shouldn't be an issue saving any of the others blank. I tested with a blank date and validation wasn't an issue, but got an error on the save request, probably something we want to look into separately

Reply:
It looks like allowing empty dates will require a small backend change, although, like Andrew said, it does seem that blank dates will ultimately work in search forms even though it shows an error when you save it.

Reply:
I updated the backend to allow empty dates.